pkg/cmd/taskrun/log_reader.go

```diff
@@ -115,6 +119,10 @@ func (lr *LogReader) readAvailableLogs(tr *v1alpha1.TaskRun) (<-chan Log, <-chan
 		podName = tr.Status.PodName
 	)
 
+	if podName == "" {
+		return nil, nil, fmt.Errorf("task %s create failed or has not hasStarted yet", lr.Task)
```

Comment:
Should read "task %s create failed or has not started yet"

Reply:
Thanks @danielhelfand  for your quickly replay. 
I noticed there is a message "task %s has not hasStarted yet"  at line 127 in the same file, do we need to update it to make consistence?

Reply:
@vincent-pli Ah, okay. I see where the confusion came from. Yes, line 127 is a typo and should be changed.